File: tengyei-backend/core-company/src/main/java/com/tengyei/company/entity/Company.java
package com.tengyei.company.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.tengyei.common.entity.BaseEntity;
import lombok.Getter;
import lombok.Setter;

import java.time.LocalDate;

@Getter
@Setter
@TableName("company")
public class Company extends BaseEntity {
    private String companyNo;
    private String fullName;
    private String shortName;
    private String creditCode;
    private String logoUrl;
    private String adminName;
    private String adminPhone;
    private String adminEmail;
    /** 0=待激活 1=启用 2=停用 */
    private Integer status;
    private LocalDate expireDate;
    private Integer maxUsers;
    private Integer maxBranches;
    private String remark;
}

File: tengyei-frontend/src/types/company.ts
export interface CompanyVO {
  id: number
  companyNo: string
  fullName: string
  shortName: string
  creditCode?: string
  adminName: string
  adminPhone: string
  adminEmail?: string
  adminUsername?: string
  remark?: string
  status: number
  expireDate?: string
  logoUrl?: string
  createdAt: string
}

export interface CompanyCreateDTO {
  fullName: string
  shortName: string
  creditCode?: string
  expireDate?: string
  adminName: string
  adminPhone: string
  adminEmail?: string
  adminUsername: string
  adminPassword: string
}

export interface CompanyUpdateDTO {
  fullName: string
  shortName: string
  creditCode?: string
  adminName?: string
  adminPhone?: string
  adminEmail?: string
  expireDate?: string
  remark?: string
  logoUrl?: string
}
